Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx

```diff
@@ -261,7 +261,7 @@ export function CreateOptions(props: CreateOptionsProps) {
         onDismiss={onDismiss}
       >
         <ChoiceGroup
-          label={formatMessage('Choose how to create your bot')}
+          // label={formatMessage('Choose how to create your bot')}
```

Comment:
prefer just remove it if we don't need it. git will remember it if we need it back later

Reply:
No problem!